src/assignment/helpers/duplicate-assignment.ts

```diff
@@ -6,6 +6,7 @@ import { ToastService } from '../../shared/services/toast-service';
 import { AssignmentService } from '../assignment.service';
 
 export async function duplicateAssignment(
+	user: Avo.User.User,
 	assignment?: Avo.Assignment.Assignment
 ): Promise<Avo.Assignment.Assignment | undefined> {
```

Comment:
Reverse orde seems more logical
first the assignment you want to duplicate, then any other info that is needed
```suggestion
export async function duplicateAssignment(
	assignment?: Avo.Assignment.Assignment,
	user: Avo.User.User
): Promise<Avo.Assignment.Assignment | undefined> {
```

Reply:
Typescript doesn't allow this because it orders paramaters from unconditional to conditional